lib/ui/window/window.cc

```diff
@@ -33,6 +35,24 @@ void Render(Dart_NativeArguments args) {
   UIDartState::Current()->window()->client()->Render(scene);
 }
 
+void SendPlatformMessage(Dart_Handle window,
+                         const std::string& name,
+                         Dart_Handle callback,
+                         const tonic::DartByteData& data) {
+  UIDartState* dart_state = UIDartState::Current();
+  blink::PlatformMessageSink sink = dart_state->platform_message_sink();
+  if (!sink)
+    return;
```

Comment:
Yeah, we should either call the callback with some sort of error or assert that this doesn't happen...